MovieDB/AppDelegate.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  AppDelegate.swift
+//  MovieDB
+//
+//  Created by Bùi Xuân Huy on 10/23/19.
+//  Copyright © 2019 huy. All rights reserved.
+//
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+        bindViewModel()
```

Comment:
em chưa khỏi tạo UIWindow mà đã bindng 
```suggestion
        window = UIWindow(frame: UIScreen.main.bounds)
        bindViewModel()
```